src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,21 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
let's also add @JoinTable with all the needed info for extra table with relations